tests/unit/components/createDialog.ts

```diff
@@ -0,0 +1,113 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import { VNode } from '@dojo/interfaces/vdom';
+import createDialog from '../../../src/components/dialog/createDialog';
+
+registerSuite({
+	name: 'createDialog',
+
+	construction() {
```

Comment:
Can we make these test names a bit clearer please? ie. `Should construct a dialog with passed properties`.